support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/ticket/refreshtoken/OAuth20DefaultRefreshTokenFactory.java

```diff
@@ -45,9 +45,15 @@ public class OAuth20DefaultRefreshTokenFactory implements OAuth20RefreshTokenFac
      */
     protected final ServicesManager servicesManager;
 
+    /**
```

Comment:
This comment is incorrect; this is not strictly what this setting does.

Your comment would also break javadocs.